presto-hive/src/test/java/io/prestosql/plugin/hive/TestOriginalFilesUtils.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive;
+
+import io.prestosql.orc.OrcReaderOptions;
+import io.prestosql.plugin.hive.orc.OriginalFilesUtils;
+import io.prestosql.spi.connector.ConnectorSession;
+import io.prestosql.testing.TestingConnectorSession;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.mapred.JobConf;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.prestosql.plugin.hive.AcidInfo.OriginalFileInfo;
+import static io.prestosql.plugin.hive.HiveTestUtils.HDFS_ENVIRONMENT;
+import static org.testng.Assert.assertTrue;
+
+public class TestOriginalFilesUtils
+{
+    private static final String tablePath = Thread.currentThread().getContextClassLoader().getResource("dummy_id_data_orc").getPath();
+
+    private static final Configuration config = new JobConf(new Configuration(false));
```

Comment:
move `new JobConf(new Configuration(false))` to `setup()` where you actualy set up this object

Reply:
fixed.